android/app/src/test/kotlin/com/vnsmap/app/routing/GraphHopperIntegrationBenchmarkTest.kt

```
@@ -223,6 +249,61 @@ class GraphHopperIntegrationBenchmarkTest {
+                    originalLat = lat,
+                    originalLon = lon,
+                    snappedLat = lat + 0.00005,
+                    snappedLon = lon + 0.00005,
+                    streetName = "Tràng Thi",
+                    distanceToRoad = 2.1,
+                    edgeId = 456,
+                    calculationTimeMs = 1L
+                )
+            }
+
+            override fun close() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)